File: progra3-0583-equipo5-frontend/Frontend_C#/SoftPac/SoftPacPersistance/DAO/EntidadesBancariasDAO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using SoftPac.Model;

namespace SoftPac.Persistance.DAO
{
    public interface EntidadesBancariasDAO
    {
        int Insertar(EntidadesBancariasDTO entidadBancaria);
        int Modificar(EntidadesBancariasDTO entidadBancaria);
        int Eliminar(EntidadesBancariasDTO entidadBancaria);
        EntidadesBancariasDTO ObtenerPorId(int entidadBancariaId);
        IList<EntidadesBancariasDTO> ListarTodos();
    }
}

File: Backend_Java/SoftPacPersistance/src/main/java/pe/edu/pucp/softpac/dao/EntidadesBancariasDAO.java
package pe.edu.pucp.softpac.dao;

import java.util.List;
import pe.edu.pucp.softpac.model.EntidadesBancariasDTO;

public interface EntidadesBancariasDAO {
    public Integer insertar(EntidadesBancariasDTO entidadBancaria);
    
    public EntidadesBancariasDTO obtenerPorId(Integer entidadBancariaId);
    
    public List<EntidadesBancariasDTO> listarTodos();
    
    public Integer modificar(EntidadesBancariasDTO entidadBancaria);
    
    public Integer eliminar(EntidadesBancariasDTO entidadBancaria);
    
    public EntidadesBancariasDTO obtenerPorNombre(String nombre);
}
